Extension Error: Cannot read property 'children' of undefined

**Lighthouse Version**: 1.4.1
**Chrome Version**: 55.0.2883.87
**URL**: http://10.1.10.7:8011/h_home.apw
**Error Message**: Cannot read property 'children' of undefined
**Stack Trace**:
 ```TypeError: Cannot read property 'children' of undefined
    at artifacts.requestCriticalRequestChains.then.chains (chrome-extension://blipmdconlkpinefehnmjammfjpmpbjk/scripts/lighthouse-background.js:560:62)```

+1 trying to use lighthouse cli


**Lighthouse Version:** 1.4.1

```
ChromeLauncher Killing all Chrome Instances +2ms
Runtime error encountered: TypeError: Cannot read property 'children' of undefined
    at artifacts.requestCriticalRequestChains.then.chains (/node_modules/lighthouse/lighthouse-core/audits/critical-request-chains.js:68:62)
TypeError: Cannot read property 'children' of undefined
    at artifacts.requestCriticalRequestChains.then.chains (/node_modules/lighthouse/lighthouse-core/audits/critical-request-chains.js:68:62)
```

at 

```
npm ERR! Darwin 16.3.0
npm ERR! node v7.1.0
npm ERR! npm  v3.10.9
```

--- a/lighthouse-core/audits/critical-request-chains.js
+++ b/lighthouse-core/audits/critical-request-chains.js
@@ -64,8 +64,8 @@ class CriticalRequestChains extends Audit {
       }
 
       // Account for initial navigation
-      const initialNavigationKey = Object.keys(chains)[0];
-      const initialNavChildren = chains[initialNavigationKey].children;
+      const initialNavKey = Object.keys(chains)[0];
+      const initialNavChildren = initialNavKey && chains[initialNavKey].children;
       if (initialNavChildren && Object.keys(initialNavChildren).length > 0) {
         walk(initialNavChildren, 0);
       }

--- a/lighthouse-core/test/audits/critical-request-chains-test.js
+++ b/lighthouse-core/test/audits/critical-request-chains-test.js
@@ -86,6 +86,8 @@ const PASSING_REQUEST_CHAIN = {
   },
 };
 
+const EMPTY_REQUEST_CHAIN = {};
+
 const mockArtifacts = (mockChain) => {
   return {
     networkRecords: {
@@ -115,6 +117,13 @@ describe('Performance: critical-request-chains audit', () => {
 
   it('calculates the correct chain result for passing example', () => {
     return Audit.audit(mockArtifacts(PASSING_REQUEST_CHAIN)).then(output => {
+      assert.equal(output.displayValue, 0);
+      assert.equal(output.score, true);
+    });
+  });
+
+  it('calculates the correct chain result for empty example', () => {
+    return Audit.audit(mockArtifacts(EMPTY_REQUEST_CHAIN)).then(output => {
       assert.equal(output.displayValue, 0);
       assert.equal(output.score, true);
     });

